WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/ShippingLineDetails.swift

```diff
@@ -111,7 +110,8 @@ private extension ShippingLineDetails {
         static let addShipping = NSLocalizedString("Add Shipping", comment: "Title for the Shipping Line screen during order creation")
         static let shipping = NSLocalizedString("Shipping", comment: "Title for the Shipping Line Details screen during order creation")
 
-        static let amountField = NSLocalizedString("Amount", comment: "Title for the amount field on the Shipping Line Details screen during order creation")
+        static let amountField = NSLocalizedString("Amount (%1$@)", comment: "Title for the amount field on the Shipping Details screen during order creation"
+                                                   + "Parameters: %1$@ - fee type (percent sign or currency symbol)")
```

Comment:
nit: we don't have percentage option here
```suggestion
                                                   + "Parameters: %1$@ - currency symbol")
```